portfolio/src/main/java/com/google/sps/servlets/DeleteComment.java

```diff
@@ -34,14 +34,15 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
-    deleteComment();
+    String kinds = request.getParameter("kind");
+
+    deleteComment(kinds);
 
-    response.sendRedirect("/comments.html");
   }
 
-  private void deleteComment() {
+  private void deleteComment(String kinds) {
```

Comment:
Are these still comments or are they something else now?

Reply:
It's a parameter for determining what type of content to delete. I will show you how it works in the demo tomorrow.